pkg/util/env_resolver.go

```diff
@@ -77,6 +77,15 @@ func GetClusterObjectNamespace() (string, error) {
 	return strData, nil
 }
 
+// GetPodNamespace returns the namespace for the pod
+func GetPodNamespace() string {
```

Comment:
The same functionality is implemented in `adapter/main.go`, that one should use this common funcion as well.

Reply:
nice catch! updated :)